api/src/test/java/ca/bc/gov/educ/api/gradstudent/service/GraduationStatusServiceTest.java

```diff
@@ -2230,7 +2230,8 @@ public void testSaveStudentRecord_DistributionRun_2() {
 
         when(graduationStatusRepository.findById(studentID)).thenReturn(Optional.of(graduationStatusEntity));
```

Comment:
This is not a real unit test. It only asserts that graduationStatusEntity is not null.

Reply:
Fixed